refract-spec.md

```diff
@@ -30,14 +30,10 @@ The Refract Element contains three properties, `element`, `attributes`, and `con
         - (string)
         - (Element)
 
-- `attributes` (enum)
+- `attributes` (object)
 
     The `attributes` property defines attributes about the given instance of the element, as specified by the `element` property. It MAY be an array of elements, or it MAY be an object. The keys of the object SHOULD be defined element types with valid values for the corresponding element types.
```

Comment:
I think you should mention the reserved properties of attributes here with links to their pertinent definitions, id, namspaces and prefix, etc.